docs/.vitepress/versions.ts

```diff
@@ -1,7 +1,25 @@
+import { execSync } from 'child_process';
```

Comment:
I think `child_process` is one of the core node packages
If possible it is now and always recommended to use the node prefix in such cases

```suggestion
import { execSync } from 'node:child_process';
```